api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,92 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot Password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot Password operation to receive an email with a password reset token.
+
```

Comment:
Suggested revision:

Use the Forgot password operation to get a password reset token delivered to the email address associated with your account.